docs/outlook/append-on-send.md

```diff
@@ -0,0 +1,192 @@
+            <bt:String id="ActionButton.Label" DefaultValue="Perform an action"/>
+          </bt:ShortStrings>
+          <bt:LongStrings>
+            <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Opens a pane displaying all available properties."/>
+            <bt:String id="ActionButton.Tooltip" DefaultValue="Perform an action when clicked."/>
+          </bt:LongStrings>
+        </Resources>
+        <ExtendedPermissions>
+          <ExtendedPermission>AppendOnSend</ExtendedPermission>
+        </ExtendedPermissions>
+      </VersionOverrides>
+    </VersionOverrides>
+    ```
+
+> [!TIP]
+> To learn more about manifests for Outlook add-ins, see [Outlook add-in manifests](manifests.md).
+
+## Implement append-on-send handling
+
+Next, implement appending on the send event.
+
+For this scenario, you'll implement appending a disclaimer to the item when the user sends.
+
+1. From the same quick start project, open the file **./src/commands/commands.js** in your code editor.
+
+1. After the `action` function, insert the following JavaScript function.
+
+    ```js
+    function appendDisclaimerOnSend(event) {
+      var appendText =
+        '<p style = "color:blue"> <i>This and subsequent emails on the same topic are for discussion and information purposes only. Only those matters set out in a fully executed agreement are legally binding. This email may contain confidential information and should not be shared with any third party without the prior written agreement of Contoso. If you are not the intended recipient, take no action and contact the sender immediately.<br><br>Contoso Limited (company number 01624297) is a company registered in England and Wales whose registered office is at Contoso Campus, Thames Valley Park, Reading RG6 1WG</i></p>';  
+      /**
+        *************************************************************
+         Ideal Usage - Call the getBodyType API. Use the coercionType
+         it returns as the parameter value below.
+        *************************************************************
+      */
+      Office.context.mailbox.item.body.appendOnSendAsync(
+        appendText,
+        {
+          coercionType: Office.CoercionType.Html
+        },
+        function(asyncResult) {
+          console.log(asyncResult);
+        }
+      );
+
+      event.completed();
+    }
+    ```
+
+1. At the end of the file, add the following statement.
+
+    ```js
+    g.appendDisclaimerOnSend = appendDisclaimerOnSend;
+    ```
+
+## Try it out
+
+1. Run the following command in the root directory of your project. When you run this command, the local web server will start (if it's not already running).
+
+    ```command&nbsp;line
+    npm run dev-server
+    ```
+
+1. Follow the instructions in [Sideload Outlook add-ins for testing](sideload-outlook-add-ins-for-testing.md) to sideload the add-in in Outlook.
+
+1. In Outlook, create a new message and add yourself to the **To** line.
```

Comment:
```suggestion
1. Create a new message, and add yourself to the **To** line.
```
While technically you do not need the comma before the conjunction, for procedures we add them more aggressively between clauses to assist machine localization.